proto/one/operations/model/external/delphi_model_execution.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+import "enum_delphi_model_mode.proto";
+import "delphi_execution_parameter.proto";
+import "delphi_configuration_execution.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.External";
+
+message DelphiModelExecution {
+    // Schema
+    string schema = 1;
+    // Version
+    string version = 2;
+    // Twin reference id associated with location or operation being modeled
+    string twinRefId = 3;
+    // Model id
+    string externalModelId = 4;
+    // Tenant id
+    string tenantId = 5;
+    // Length of forecast in hours
+    double forecastDurationHours = 6;
+    // Timestamps of the provided input data
+    repeated string timestamps = 7;
+    // Current inputs to the model
+    repeated DelphiExecutionParameter inputs = 8;
+    // Current outputs for the model
+    repeated DelphiExecutionParameter outputs = 9;
+    //list of supported Delphi model modes
+    repeated EnumDelphiModelMode modelModes = 10;
```

Comment:
would change to modelModelsEnabled just to be clearer about what this is doing.

Reply:
Would that be just for the top level field or also for the ones on each parameter? Just to clarify you are requesting that the top level field be `modelModesEnabled` correct?  You comment said modelMode**l**sEnabled so wanted to be sure.

Reply:
I think he wants me to change the modelModes to modelModesEnabled

Reply:
sorry should have added more details.  this field means slightly different things in the different protobuf models.  Here it is indicating which features we are *enabling* (nowcast, forecast, optimization).  In the other protobufs it indicates that a particular configuration, input, or output *applies* to one of the features (nowcast, forecast, optimization).  I think we want names that described that.  So yes we should change the name as you have done here to modelModelsEnabled, but in the other protobufs I think we should just leave it as modelModels (even though the type is the same).  So I think the renaming needs to undo on the other protobufs.  Hope that makes sense

Reply:
@josephquiroz reach out if you need more clarification